data/actions/actions.xml

```diff
@@ -42,6 +42,9 @@
 	<!-- Farmine -->
 	<action fromaid="9118" toaid="9121" script="farmine/elevatorLever.lua" />
 
+	<!-- Temple of Equilibrium -->
```

Comment:
just add the folder normally in windows and in git shell you write:

git add

and press "tab" until the right folder appears and press enter